Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "gateway_config"
-version = "0.2.2"
+version = "0.2.3"
```

Comment:
There's no stated motivation for this change. Why is this necessary? If not, why is this part of the PR?

Reply:
I will put it in a commit message, but we have added features to the crate and if we don't change the version in the user (and the provider) it is possible to have a mismatch, this makes sure that doesn't happen.

Reply:
Commit modified to explain reasoning.